includes/downloads/recalculations.php

```diff
@@ -0,0 +1,120 @@
+<?php
+/**
+ * Recalculation functions for downloads.
+ * @package     EDD
+ * @subpackage  Downloads
+ * @copyright   Copyright (c) 2022, Easy Digital Downloads, LLC
+ * @license     http://opensource.org/licenses/gpl-2.0.php GNU Public License
+ * @since       3.1
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * The hook is registered here because it's specifically for the cron job,
+ * but it calls the general download recalculation function.
```

Comment:
Let's add a `since` tag on this one, so we can know when it was added in the future.